bot/.env.example

```
@@ -0,0 +1,16 @@
+ENV=dev
+PORT=8787
+SPOT_API_URL=https://testnet.binance.vision
+SPOT_STREAM_URL=wss://testnet.binance.vision/ws
+FUT_API_URL=https://testnet.binancefuture.com
+FUT_WSS_URL=wss://testnet.binancefuture.com/ws-fapi/v1
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The FUT_WSS_URL key should go before the PORT key

(UnorderedKey)